contracts/DefaultStrategy.sol

```diff
@@ -522,248 +443,289 @@ contract DefaultStrategy is UUPSOwnableUpgradeable, UsingRegistryUpgradeable, Ma
     }
 
     /**
-     * @notice Returns the active groups that can receive the entire `votes`
-     * amount based on their current receivable votes limit in Election.sol.
-     * @param votes The number of votes that would potentially be added.
-     * @return The list of votable active groups.
+     * @notice Rebalances CELO between groups that have incorrect CELO-stCELO ratio.
+     * FromGroup is required to have more CELO than it should and ToGroup needs
+     * to have less CELO than it should.
+     * @param fromGroup The from group.
+     * @param toGroup The to group.
      */
-    function getVotableGroups(uint256 votes) internal returns (address[] memory) {
-        uint256 numberGroups = activeGroups.length();
-        uint256 numberVotableGroups = 0;
-        address[] memory votableGroups = new address[](numberGroups);
+    function rebalance(address fromGroup, address toGroup) public {
```

Comment:
What is the difference between `manager.rebalance` and `defaultStrategy.rebalance`?

Add an explanation in the natspec.

Reply:
The idea was that in this way noone can really gamify the default strategy. Also this can help when one of the groups would be overflowing or close to overflow.